src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -110,7 +111,8 @@ public function assertExists($path, $content = null)
 
         foreach ($paths as $path) {
             PHPUnit::assertTrue(
-                $this->exists($path), "Unable to find a file or directory at path [{$path}]."
+                $this->exists($path),
+                "Unable to find a file or directory at path [{$path}]."
```

Comment:
Can you please undo all of these formatting changes?

Reply:
Sorry about that. it was my php-cs-fixer doing this. I undid all of the formatting changes 👍